### PR TITLE
Print fava version on help index page

### PR DESCRIPTION
**Pull request by aclindsa**
Closes https://github.com/beancount/fava/issues/1181

### CODE REVIEW

**Comment by aclindsa** [comment]
I am confused by the CI failures here. It says:
> src/fava/application.py:26: error: Module 'beancount' has no attribute '__version__'

But on my local Ubuntu, importing beancount's version works just fine:

```
root@c679fc96b91c:/tmp/build/fava_investor# cat /etc/lsb-release
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=20.04
DISTRIB_CODENAME=focal
DISTRIB_DESCRIPTION="Ubuntu 20.04.2 LTS"
root@c679fc96b91c:/tmp/build/fava_investor# python3
Python 3.8.5 (default, Jan 27 2021, 15:41:15)
[GCC 9.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from beancount import __version__ as beancount_version
>>> print(beancount_version)
2.3.4
```

@yagebu Is there something special going on with tox/mypy's environment or something that I'm failing to comprehend?

**Comment by aclindsa** [comment]
> > src/fava/application.py:26: error: Module 'beancount' has no attribute '**version**'

To be clear, this was originally formatted as `__version__` in the error message.

**Comment by aclindsa** [comment]
> @yagebu Is there something special going on with tox/mypy's environment or something that I'm failing to comprehend?

I'm still not sure I understand *why*, but importing top-level `beancount` instead of trying to selectively import `beancount.__version__` allows the 'mypy' check to succeed. I've updated this PR to use that instead for now, pending your input.

File: src/fava/help/_index.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
-Welcome to the help pages for Fava! There are pages for the following topics:
+Welcome to the help pages for Fava! You are running beancount version
```
Comment (yagebu) [inline src/fava/help/_index.md:1]: Beancount

File: src/fava/help/_index.md [hunk 2]
```diff
@@ -1,4 +1,7 @@
...
+`{{ beancount_version }}` and fava `{{ fava_version }}`
```
Comment (yagebu) [inline src/fava/help/_index.md:2]: Fava

**Review by yagebu — CHANGES_REQUESTED** [review]
Thanks for the PR! :)

The types for Beancount are not coming from Beancount, but rather the stub type files in the stubs directory. If you add the version to `stubs/beancount/init.pyi`, the import should work directly.

Since this adds some dynamic content to the help pages, it would be good to add a test that the help index page renders without errors and that it inserts the version numbers.

**Comment by aclindsa** [comment]
> The types for Beancount are not coming from Beancount, but rather the stub type files in the stubs directory. If you add the version to `stubs/beancount/init.pyi`, the import should work directly.

Ah, doing that does allow me to use the more selective import. Thanks for the tip!

> Since this adds some dynamic content to the help pages, it would be good to add a test that the help index page renders without errors and that it inserts the version numbers.

Done! I also fixed the capitalization on 'Fava' and 'Beancount' you noted.

**Review by yagebu — APPROVED** [review]
Thanks again, this looks great :)

**Pull request merged**